src/constants.js

```diff
@@ -13,7 +13,7 @@ export const DEPENDENCY_TYPES = ['devDependencies', 'dependencies', 'optionalDep
 export const RESOLUTIONS = 'resolutions';
 export const MANIFEST_FIELDS = [RESOLUTIONS, ...DEPENDENCY_TYPES];
 
-export const SUPPORTED_NODE_VERSIONS = '^4.8.0 || ^5.7.0 || ^6.2.2 || ^8.0.0';
+export const SUPPORTED_NODE_VERSIONS = '^4.8.0 || ^5.7.0 || ^6.2.2 || ^8.0.0 || ^9.0.0';
```

Comment:
I think we can simply do `>= 8.0.0` for now to be more future-proof.

Reply:
Ok, but then we'd need to add that to CI

Reply:
Not really. It is just to make the warning go away since we simply don't know.